### PR TITLE
Eliminate unnecessary echo and cut commands

### PR DESCRIPTION
**Pull request by framps**
I don't know whether performance is an issue for you. I detected you use cat and echo in your code when it's actually not needed and creates some runtime overhead.

My PR removes the usage of these commands. I also added another way to extract the syno token with sed instead with mutliple cut commands. The code is commented out for your reference. Not sure whether you like regex.

I'm not able to test and verify my changes on a Synology. But I think they are straightforward.

### CODE REVIEW

File: package/ui/index.cgi [hunk 1]
```diff
@@ -36,16 +36,16 @@ job_version="0.6-000"
 
 	app_name="BasicBackup"
 	app_title="Basic Backup"
-	app_home=$(echo /volume*/@appstore/${app_name}/ui)
-	app_link=$(echo /webman/3rdparty/${app_name})
+	app_home="/volume*/@appstore/${app_name}/ui"
```
Comment (geimist) [inline package/ui/index.cgi]: A masked path with a wildcard will probably not be resolved correctly!

```suggestion
	app_home=$(echo /volume*/@appstore/${app_name}/ui)
```

Comment (framps) [reply package/ui/index.cgi]: > A masked path with a wildcard will probably not be resolved correctly!

Not sure I get your point. Would you please give an example?

Comment (geimist) [reply package/ui/index.cgi]:
```
geimist@DS:~$ app_name="BasicBackup"
geimist@DS:~$ app_home="/volume*/@appstore/${app_name}/ui"
geimist@DS:~$ [ -d "$app_home" ] && echo "is a directory" || echo "is not a directory"
is not a directory

geimist@DS:~$ app_home=$(echo /volume*/@appstore/${app_name}/ui)
geimist@DS:~$ [ -d "$app_home" ] && echo "is a directory" || echo "is not a directory"
is a directory
```

Comment (framps) [reply package/ui/index.cgi]: Good catch :+1:

**Pull request closed without merging**